tests/jerry/es2015/for-of.js

```diff
@@ -119,3 +119,109 @@ var i = 0;
 for (var a of obj) {
   assert (a === i++);
 }
+
+function __createIterableObject(arr, methods) {
+  methods = methods || {};
```

Comment:
You can use a default parameter

Reply:
This method comes from the `kangax` test suite, therefore I left the original source.

Reply:
What about the license?

Reply:
[These methods](https://github.com/kangax/compat-table/blob/gh-pages/node.js#L29) only helper functions to run the test suite but useful for testing purposes. I see no license marker from the origin of the file.

Reply:
Beware! Here be dragons! The fact that the file has no license header doesn't mean that it is not covered by a license. (If it had no license at all, that would be the most dangerous situation, actually. That would disallow its reuse anywhere!)

Fortunately, the project repo does contain a [license](https://github.com/kangax/compat-table/blob/gh-pages/LICENSE), which covers all files, including the source reused here. One small problem remains: the git logs claim that this is the MIT license, but actually it isn't: it is some kind of MIT--. The MIT license would contain the condition "The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software", but that's missing from this license. 

Nevertheless, I think it is safe to assume the unmodified MIT license as that's more strict. But then, we should replicate the MIT license in our code. There is example for that, e.g., in jerry-libm.

Some background info on MIT license:
- https://opensource.org/licenses/MIT
- https://tldrlegal.com/license/mit-license

Reply:
@akosthekiss Thanks for the clarification, I've missed that part.